# HG changeset patch
# User Julien Bect <jbect@users.sourceforge.net>
# Date 1547446305 -3600
#      Mon Jan 14 07:11:45 2019 +0100
# Branch stable
# Node ID 6657213145e5664db64bb8f8a6881a343d59355a
# Parent  e590d8ad9ed01e021940e0f9f4dc5f63188851b8
libinterp/parse-tree/pt-jit.cc: comment: Fix a typo

diff --git a/libinterp/parse-tree/pt-jit.cc b/libinterp/parse-tree/pt-jit.cc
--- a/libinterp/parse-tree/pt-jit.cc
+++ b/libinterp/parse-tree/pt-jit.cc
@@ -64,17 +64,17 @@ along with Octave; see the file COPYING.
 #endif
 // In LLVM 3.7.x and earlier, we use createBasicAliasAnalysisPass
 // from llvm/Analysis/Passes.h (already included above)
 
 #if defined (HAVE_LLVM_BITCODE_READERWRITER_H)
 // In LLVM <= 3.9, only one header for bitcode read/writer
 #  include <llvm/Bitcode/ReaderWriter.h>
 #else
-// Satrting with LLVM 4.0, two separate headers
+// Starting with LLVM 4.0, two separate headers
 #  include <llvm/Bitcode/BitcodeReader.h>
 #  include <llvm/Bitcode/BitcodeWriter.h>
 #endif
 
 #include <llvm/ExecutionEngine/ExecutionEngine.h>
 // old JIT, LLVM < 3.6.0
 // #include <llvm/ExecutionEngine/JIT.h>
 // MCJIT, LLVM >= 3.0.0
